src/main/asciidoc/kubernetes-server.adoc

```diff
@@ -1,107 +1 @@
-
-NAME                  TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                      AGE
-service/kafka         ClusterIP      10.100.200.97    <none>          9092/TCP                     7m
-service/kafka-zk      ClusterIP      10.100.200.244   <none>          2181/TCP,2888/TCP,3888/TCP   7m
-service/kubernetes    ClusterIP      10.100.200.1     <none>          443/TCP                      19d
-service/mysql         ClusterIP      10.100.200.152   <none>          3306/TCP                     7m
-service/redis         ClusterIP      10.100.200.8     <none>          6379/TCP                     1m
-service/scdf-server   LoadBalancer   10.100.200.32    44.333.222.11   80:31486/TCP                 5m
-
-NAME                           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-deployment.apps/kafka-broker   1         1         1            1           7m
-deployment.apps/kafka-zk       1         1         1            1           7m
-deployment.apps/mysql          1         1         1            1           7m
-deployment.apps/redis          1         1         1            1           1m
-deployment.apps/scdf-server    1         1         1            1           5m
-
-NAME                                      DESIRED   CURRENT   READY   AGE
-replicaset.apps/kafka-broker-696786c8f7   1         1         1       7m
-replicaset.apps/kafka-zk-5f9bff7d5        1         1         1       7m
-replicaset.apps/mysql-f878678df           1         1         1       7m
-replicaset.apps/redis-748db48b4f          1         1         1       1m
-replicaset.apps/scdf-server-64fb996ffb    1         1         1       5m
-----
-+
-. Connect the `shell` with `server` running on Kubernetes. 
-Note the `EXTERNAL-IP` entry for `service/scdf-server`. 
-Connect using the default username and password.
-+
-```
-$ cd <PATH/TO/SPRING-CLOUD-DATAFLOW-SHELL-JAR>
-$ java -jar spring-cloud-dataflow-shell-<VERSION>.jar
-
-  ____                              ____ _                __
- / ___| _ __  _ __(_)_ __   __ _   / ___| | ___  _   _  __| |
- \___ \| '_ \| '__| | '_ \ / _` | | |   | |/ _ \| | | |/ _` |
-  ___) | |_) | |  | | | | | (_| | | |___| | (_) | |_| | (_| |
- |____/| .__/|_|  |_|_| |_|\__, |  \____|_|\___/ \__,_|\__,_|
-  ____ |_|    _          __|___/                 __________
- |  _ \  __ _| |_ __ _  |  ___| | _____      __  \ \ \ \ \ \
- | | | |/ _` | __/ _` | | |_  | |/ _ \ \ /\ / /   \ \ \ \ \ \
- | |_| | (_| | || (_| | |  _| | | (_) \ V  V /    / / / / / /
- |____/ \__,_|\__\__,_| |_|   |_|\___/ \_/\_/    /_/_/_/_/_/
-
-
-Welcome to the Spring Cloud Data Flow shell. For assistance hit TAB or type "help".
-server-unknown:>
-```
-+
-```
-server-unknown:>dataflow config server --uri http://<SCDF_EXTERNAL_IP> --username user --password password
-...
-dataflow:>
-```
-
+Follow the https://dataflow.spring.io/docs/installation/kubernetes/[installation] instructions to run Spring Cloud Data Flow on Kubernetes.
```

Comment:
Since now we point to the installation in Microsite, which is great, but it does the break at the username/password setup for the file-ingest demo. Maybe we still need this section only for this sample?

Reply:
It depends on the installation. The helm chart currently has 

```
SPRING_DATASOURCE_USERNAME=root
SPRING_DATASOURCE_PASSWORD=rootpw`
```

Reply:
what breaks?

Reply:
No, I meant the basic-auth configuration in SCDF itself. 

For example, in the stream definition, you point to `--spring.cloud.dataflow.client.authentication.basic.username=user` and `--spring.cloud.dataflow.client.authentication.basic.password=password` as inline properties. That means, SCDF is protected by basic-auth. If that is true, when you logon to Shell/UI, you'd have to pass the user/pass to authenticate also.

The install instructtions in Microsite don't speak to user/pass.

You'd need something like: 

> server-unknown:>dataflow config server --uri http://<SCDF_EXTERNAL_IP> --username user --password password

Which is what you're replacing here with Microsite links. 

I hope this clarifies it.

Reply:
done